_posts/2019-11-26-flyway-6.1.0.md

```diff
@@ -39,7 +39,7 @@ Find out more [here](/documentation/database/snowflake)!
 
 ## Machine-readable output
 
-You can now output JSON from `flyway info` by using the `-json` flag.
+You can now output JSON from `flyway info` by using the `-json` flag (note: superseded by `-outputType=json` in Flyway 7).
```

Comment:
We haven't been doing versioned documentation. In all other cases we just delete the old thing (in this case, delete all references to -json entirely).

Reply:
True...